examples/http_request_interceptor/http_request_interceptor.bal

```diff
@@ -0,0 +1,70 @@
+import ballerina/http;
+import ballerina/io;
+
+// Header name to be set to the request in the request interceptor.
+final string interceptor_header1 = "X-requestHeader1";
+final string interceptor_header2 = "X-requestHeader2";
+
+// Header value to be set to the request in the request interceptor.
+final string interceptor_header_value1 = "RequestInterceptor1";
+final string interceptor_header_value2 = "RequestInterceptor2";
+
+// A Request Interceptor service class implementation. It intercepts the Request and adds a header before it dispatched to the 
+// target HTTP Resource. A Request Interceptor service class can have only one resource function.
+service class RequestInterceptor1 {
+    *http:RequestInterceptor;
+
+    // A default resource function which will be executed for all requests. `RequestContext` is used to share data between 
+    // interceptors. Resource methods are only allowed to return `http:NextService|error?`
+    resource function 'default [string... path](http:RequestContext ctx, 
+                            http:Request req) returns http:NextService|error? {
+        io:println("Executing Request Interceptor 1");
+        // Sets a header to the request inside the interceptor service.
+        req.setHeader(interceptor_header1, interceptor_header_value1);
+        // Returns the next interceptor or the target service in the pipeline. An error is returned when the call fails.
+        return ctx.next();
+    }
+}
+
+// Creates a new Request Interceptor
+RequestInterceptor1 requestInterceptor1 = new;
+
+// Another Request Interceptor service class.
+service class RequestInterceptor2 {
+    *http:RequestInterceptor;
+
+    // This interceptor is only executed for GET requests with path "/greeting".
+    resource function get greeting(http:RequestContext ctx, http:Request req) 
+                        returns http:NextService|error? {
+        io:println("Executing Request Interceptor 2");
```

Comment:
I think above also we can remove.

Reply:
Removed in [`1ee1b04`](https://github.com/ballerina-platform/ballerina-distribution/pull/2365/commits/1ee1b04fe7175614fd9007a2291fe598b2b3debb)